src/scene_battle_rpg2k3.cpp

```diff
@@ -383,6 +383,15 @@ void Scene_Battle_Rpg2k3::CreateUi() {
 	} else {
 		SetupSystem2Graphics();
 	}
+
+	std::vector<Game_Battler*> battlers;
+	Main_Data::game_party->GetActiveBattlers(battlers);
+	for (std::vector<Game_Battler*>::const_iterator it = battlers.begin(); it != battlers.end(); ++it) {
```

Comment:
Use a C++11 for each loop here and ditch the iterators.

Reply:
Did the requested change as well.